features/citizens/citizens.feature

```diff
@@ -85,7 +85,8 @@ Feature: Citizen journey
 
   @javascript
   Scenario: I want to change property details via the check your answers page
-    Given I complete the citizen journey as far as check your answers
+    Given I have completed an application
+    And I complete the citizen journey as far as check your answers
```

Comment:
Done in two steps so that subsequent steps can modify the created application before stepping into the page.